From 869d5a2d56d4ab240568fa5fd54ae3b792d9e3c5 Mon Sep 17 00:00:00 2001
From: Yazan Shhady <yazan.shhady@solid-run.com>
Date: Tue, 21 Nov 2023 13:08:21 +0200
Subject: [PATCH] imx8mn-compact: update the adxl345 sensor i2c address

- SDO/SAO pin in ADXL345 get value 0 (R165 is assembled)
- Update the ADXL34X I2C Device Address to 0x53

Selctable ADXL34X I2C Device Address:
	- SDO/SA0=0 -> I2C Device Address=0x53
        - SDO/SA0=1 -> I2C Device Address=0x1d

Signed-off-by: Yazan Shhady <yazan.shhady@solid-run.com>
---
 arch/arm64/boot/dts/freescale/imx8mn-compact.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mn-compact.dts b/arch/arm64/boot/dts/freescale/imx8mn-compact.dts
index a92a01bbdfca..e2b8c17acb2a 100755
--- a/arch/arm64/boot/dts/freescale/imx8mn-compact.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mn-compact.dts
@@ -527,9 +527,9 @@
 	sda-gpios = <&gpio5 19 GPIO_ACTIVE_HIGH>;
 	status = "okay";
 
-	accelerometer@1d {
+	accelerometer@53 {
 		compatible = "adi,adxl345";
-		reg = <0x1d>;
+		reg = <0x53>;
 		interrupt-parent = <&gpio1>; // GPIO1_IO09
 		interrupts = <9 IRQ_TYPE_LEVEL_HIGH>;
 	};
-- 
2.39.2

